### PR TITLE
Flesh out paging introduction

### PR DESCRIPTION
**Pull request by rylev**
- flesh out more on an immediate problem paging solves
- hint that paging is used to implement virtual memory and that virtual memory is good for other things we'll see in the future

@steveklabnik @LaylConway what do you think?

### CODE REVIEW

**Comment by LaylBongers** [comment]
It's indeed much clearer like this :+1:

File: src/paging.md [hunk 1]
```diff
@@ -71,6 +80,16 @@ to keep track of literally every memory address and where it points to.
 Instead, we split up memory into chunks, also called ‘pages’, and then map each
 page to an equal sized chunk of physical memory.
 
+> **By the way...**
+> We're using paging to implement something called "virtual memory". Besides
```
Comment (steveklabnik) [inline src/paging.md]: We aren't yet; we will be someday, once we have user mode

**Comment by steveklabnik** [comment]
A nit, but agreed, looks good overall :+1:

**Comment by rylev** [comment]
@steveklabnik this should be ready to go now

**Comment by steveklabnik** [comment]
Looks great, thanks!

**Pull request merged**